app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :groups
+  has_many :movements
+
+  validates :name, presence: true, allow_blank: false
```

Comment:
- [ ] Kindly add validation for all user input to make sure that anyone with bad intentions cannot compromise their app. eg: `email, password, ...`

Reply:
@BenMukebo Ohhh you are absolutely right, I had forgotten to put them but thank you very much for letting me know. I've already given all validations a review as well to confirm that I hadn't missed another one.

Reply:
This change has been done well

Reply:
@peterrobert Thank you